docs/sources/getting-started/glossary.md

```diff
@@ -28,6 +28,12 @@ This topic lists words and abbreviations that are commonly used in the Grafana d
       A file, database, or service providing the data. Grafana supports a several data sources by default, and can be extended to support additional ones through plugins.
     </td>
   </tr>
+  <tr>
+    <td style="vertical-align: top">Exemplar</td>
+    <td>
+      An exemplar is any data that serves as a detailed example of data used to create an aggregation. A common type of examplars is a trace that references an aggregated metric.
```

Comment:
Some small suggestions.

"An exemplar is any data that " => "An exemplar is a piece of metadata that ... "

"type of examplars is a trace that references an aggregated metric" => "type of exemplars is a trace id that exemplifies a metric"

Reply:
I think "piece of" is redundant, what about just "An examplar is metadata that ..."?

The latter suggestion, implies that an examplar is the _reference_ to the trace, rather the actual trace? Is this intended?

Reply:
> I think "piece of" is redundant, what about just "An examplar is metadata that ..."

Agree!

> The latter suggestion, implies that an examplar is the reference to the trace, rather the actual trace? Is this intended?

Yes.  The metadata attached is only a trace id.  The trace itself would need to be stored in a different system.

Also, heads up that you have misspelled "exemplars" as "examplars" in one spot.

Reply:
How about this then?:

> An exemplar is metadata that references a detailed example of data used to create an aggregation. A common type of exemplar is an ID that connects an aggregated metric with a trace for one of the request used in the aggregation.

Reply:
looks good to me!